specification/http/1.0/openapi.json

```diff
@@ -0,0 +1,456 @@
+                },
+                "user_info": {
+                  "type": "object",
+                  "description": "Additional information about the user, if present."
+                }
+              },
+              "additionalProperties": false
+            }
+          },
+          "required": ["execution_id", "tool_id"],
+          "additionalProperties": false
+        },
+        "ExecuteToolResponse": {
+          "type": "object",
+          "properties": {
+            "$schema": {
+              "type": "string",
+              "format": "uri"
+            },
+            "execution_id": {
+              "type": "string",
+              "description": "The globally-unique ID for this tool execution."
+            },
+            "duration": {
+              "type": "number",
+              "description": "The duration of the tool call, in milliseconds"
+            },
+            "success": {
+              "type": "boolean",
+              "description": "Whether the tool execution was successful."
+            },
+            "output": {
+              "type": "object",
+              "oneOf": [
+                {
+                  "properties": {
+                    "value": {
+                      "description": "The value returned from the tool.",
+                      "oneOf": [
+                        {
+                          "type": "object",
+                          "additionalProperties": true
+                        },
+                        {
+                          "type": "number"
+                        },
+                        {
+                          "type": "string"
+                        },
+                        {
+                          "type": "boolean"
+                        }
+                      ]
+                    }
+                  },
+                  "required": ["value"],
+                  "additionalProperties": false
+                },
+                {
+                  "properties": {
+                    "error": {
+                      "type": "object",
+                      "properties": {
+                        "message": {
+                          "type": "string",
+                          "description": "An error message that can be shown to the user or the AI model."
+                        },
+                        "developer_message": {
+                          "type": "string",
+                          "description": "An internal message that will be logged but will not be shown to the user or the AI model"
+                        },
+                        "can_retry": {
+                          "type": "boolean",
+                          "description": "Whether the tool call can be retried by the client.",
+                          "default": false
+                        },
+                        "additional_prompt_content": {
```

Comment:
The "additional" part is vague. How does would a client use it in the retry prompt?